spec/amp-3p-video.md

```diff
@@ -0,0 +1,82 @@
+
+Some video cannot be embedded this way. AMP provides vendor-optimized components like [`amp-youtube`](https://go.amp.dev/c/amp-youtube),
+[`amp-ima-video`](https://go.amp.dev/c/amp-ima-video), amongst [others](./amp-video-interface.md).
+Under the hood, these load an iframe whose page communicates with the outer document to coordinate playback.
+
+For custom video implementations, [`amp-video-iframe`](https://go.amp.dev/c/amp-video-iframe) is a generic component that can load any video document to coordinate its playback.
+
+## I want to contribute my vendor-specific player
+
+You probaby do **_not_** need to build your own player.
+
+The `amp-video-iframe` playback interface supports the following methods, which are likely to cover all your bases:
+
+- `play`
+- `pause`
+- `mute`
+- `unmute`
+- `showcontrols`
+- `hidecontrols`
+- `fullscreenenter`
+- `fullscreenexit`
+
+It can also [send custom namespaced analytics signals](<https://amp.dev/documentation/components/amp-video-iframe/#postanalyticsevent(eventtype[,-vars])>).
+
+If this is enough for your use case, you'd only need to build a playback document hosted on your server that's able to communicate with `amp-video-iframe`.
+For guidance on accomplishing this, [refer to the component documentation.](https://go.amp.dev/c/amp-video-iframe)
+
+If you _cannot_ support these playback methods, it's likely that a simple [`amp-iframe`](https://go.amp.dev/c/amp-iframe)
+can embed a separate video document just fine.
+
+### When _should_ I build a vendor-specific player?
+
+If the API you require for communication includes methods that are **_not_** part of the `amp-video-iframe` playback interface, you might want to build your own player. (Please note that most custom analytics signals are not considered in this case,
+since `amp-video-iframe` [also has plumbing for that.](<https://amp.dev/documentation/components/amp-video-iframe/#postanalyticsevent(eventtype[,-vars])>))
+
+For example, these are some of the features that justify specific players:
+
+- **`amp-youtube`** loads initial-frame placeholder images based on a URL scheme for perceived performance.
+  Its underlying implementation also loads with lower-than-default priority, since it's rather large in binary size.
+
+- **`amp-ima-video`** embeds a generic host-less page that ships with the IMA SDK in order to transparently include standard IMA interface videos.
+
+(Please note that most other 3p players historically exist before `amp-video-iframe` could fulfill their use case.)
+
+## How can I build a player component?
+
+### Understand the contribution process
+
+You'll need to go through [the standard AMP contribution process](../CONTRIBUTING.md) when creating and submitting your component.
+
+Your player component is also shipped as an extension, so you should [become familiar with the process of building one.](https://github.com/ampproject/amphtml/blob/master/contributing/building-an-amp-extension.md)
```

Comment:
I'd recommend mentioning the I2I process and design review explicitly here.

Reply:
I described I2I but most video player components follow the same patterns and they've never had to go through design review, so I omitted that.